opensrp-opd/src/main/java/org/smartregister/opd/utils/LookUpUtils.java

```diff
@@ -0,0 +1,135 @@
+package org.smartregister.opd.utils;
+
+import android.database.Cursor;
+import android.os.AsyncTask;
+
+import org.apache.commons.lang3.StringUtils;
+import org.smartregister.AllConstants;
+import org.smartregister.Context;
+import org.smartregister.commonregistry.CommonPersonObject;
+import org.smartregister.commonregistry.CommonRepository;
+import org.smartregister.cursoradapter.SmartRegisterQueryBuilder;
+import org.smartregister.event.Listener;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import timber.log.Timber;
+
+
+public class LookUpUtils {
+    public static void lookUp(final Context context, final Map<String, String> entityLookUp, final Listener<List<CommonPersonObject>> listener, Object o) {
+        org.smartregister.util.Utils
+                .startAsyncTask(new AsyncTask<Void, Void, List<CommonPersonObject>>() {
+                    @Override
+                    protected List<CommonPersonObject> doInBackground(Void... params) {
+                        publishProgress();
+                        return clientLookUp(context, entityLookUp);
+                    }
+
+                    @Override
+                    protected void onPostExecute(List<CommonPersonObject> result) {
+                        listener.onEvent(result);
+//                        if (progressBar != null) {
+//                            progressBar.setVisibility(View.GONE);
+//                        }
+                    }
+
+                    @Override
+                    protected void onProgressUpdate(Void... values) {
+//                        if (progressBar != null) {
+//                            progressBar.setVisibility(VISIBLE);
+//                        }
+                    }
+                }, null);
+    }
```

Comment:
Reformat this file to fix the spacing issues. We currently don't have Codacy enabled but it would have been the one to check on this issues. Sorry, it has to be me